scene/3d/physics_body_3d.cpp

```diff
@@ -2998,8 +2998,8 @@ void PhysicalBone3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "linear_damp", PROPERTY_HINT_RANGE, "0,100,0.001,or_greater"), "set_linear_damp", "get_linear_damp");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "angular_damp_mode", PROPERTY_HINT_ENUM, "Combine,Replace"), "set_angular_damp_mode", "get_angular_damp_mode");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "angular_damp", PROPERTY_HINT_RANGE, "0,100,0.001,or_greater"), "set_angular_damp", "get_angular_damp");
-	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "linear_velocity"), "set_linear_velocity", "get_linear_velocity");
-	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "angular_velocity"), "set_angular_velocity", "get_angular_velocity");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "linear_velocity", PROPERTY_HINT_NONE, "suffix:m/s"), "set_linear_velocity", "get_linear_velocity");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "angular_velocity", PROPERTY_HINT_NONE, "suffix:rad/s"), "set_angular_velocity", "get_angular_velocity");
```

Comment:
This probably makes more sense like it is now with `rad/s`, but a thought that occurred to me is that we could make this degrees per second. Leaving a comment here for discussion's sake.

Reply:
That's a good idea. The range property hint might need to have support for rad/sec to deg/sec converstion just like it supports radians to degrees conversion.

Reply:
IIRC, the hint for radians to degrees conversion already allows overriding the suffix, so I think `U"radians,suffix:\u00B0/s"` (or  `"radians,suffix:deg/s"`) would do it. I haven't tested. But we don't need to do this yet, it depends on what others think.

Reply:
I changed it to `°/s`, but it looks a little odd. I'm not sure if `deg/s` is better, but I do think converting radians to degrees is better here because that's done everywhere else there are properties measured in radians.

![image](https://user-images.githubusercontent.com/67974470/169676356-4de33463-5674-486e-b503-75da1fecd2e8.png)